tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -700,7 +713,9 @@ ROOT::Experimental::Detail::RPageSourceDaos::LoadClusters(std::span<RCluster::RK
    for (unsigned i = 0; i < clusterKeys.size(); ++i) {
       const auto &clusterKey = clusterKeys[i];
       auto clusterId = clusterKey.fClusterId;
-      std::map<std::uint32_t, std::vector<RDaosSealedPageLocator>> onDiskClusterCages;
+      // Group page locators by their position in the object store; with caging enabled, this facilitates the
+      // processing of cages' requests together into a single IOV to be populated.
+      std::map<std::uint32_t, std::vector<RDaosSealedPageLocator>> onDiskClusterPages;
```

Comment:
AFAIK, order is not required and therefore we could use `std::unordered_map` instead which is O(1) amortized for insertion and lookup.